vignettes/Status.Rmd

```diff
@@ -231,9 +231,9 @@ dl_data <- readWQPdata(
 ## whatNWISdata
 
 
-NWIS services are "frozen": the returned data availability will also be frozen ONLY for "qw" data_type_cd results. All other data types should not be affected. 
+NWIS discrete water quality services are "frozen": the returned data availability will also be frozen ONLY for "qw" data_type_cd results. All other data types should not be affected. 
 
-When the NWIS services are decommissioned (possible September 2024): there will no longer be any "qw" information provided in the output of `whatNWISdata`. Discrete water-quality availability will be available via WQP services. More information will be provided as we learn more.
+When the NWIS services are decommissioned (likely in 2025): there will no longer be any "qw" information provided in the output of `whatNWISdata`. Discrete water-quality availability will be available via WQP services. More information will be provided as we learn more.
```

Comment:
Actually, I think this is incorrect. I'm wondering if `whatNWISdata` still returns older qw data, or if the service HAS been decommissioned.

Reply:
I was just looking at this. It's still showing qw data, but I'll try to get clarification on what will happen when the NWIS qw service actually gets decommissioned

Reply:
qwdata isn't slated for decommission until end of Q1, so it should still be working. Another tact for this warning is to just say that the 'qw' information has not been updated since March 2024 and will soon become deprecated: we recommend users obtain water quality data availability  from the WQP services instead. Or something

Reply:
Right, but will there be _any_ qw info from the site inventory after Q1, or will there always be the stale info?

Reply:
if that info is being pulled from an rdb then yes, that info will no longer be available. IoW will not support a public endpoint for data in the qwdata DB indefinitely, at some point it will be decommissioned.

Reply:
Yes, the data should stop flowing when qwdata is decommissioned.